01_Introduction_to_DP02.ipynb

```diff
@@ -450,7 +450,23 @@
    "id": "8ced15d7",
    "metadata": {},
    "source": [
-    "### 2.3. Retrieving data with TAP"
+    "### 2.3. Retrieving data with TAP\n",
+    "\n",
+    "Let's start with a few tips about how to do efficient queries on the DP0.2 catalogs.\n",
+    "\n",
+    "**RA, Dec constraints yeild faster queries:**\n",
+    "LSST Query Services (Qserv) provides access to the LSST Database Catalogs.\n",
+    "Users can query the catalogs using standard SQL query language with a few restrictions.\n",
+    "Qserv stores catalog data sharded by coordinate (RA, Dec).\n",
+    "ADQL query statements that include constraints by coordinate do not requre a whole-catalog search, and are typically faster (and can be much faster) than ADQL query statements which only include constraints for other columns.\n",
+    "\n",
+    "**Retrieve a small sample of rows:**\n",
+    "As demonstrated in Section 2.3.2, use `maxrec=10` or `SELECT TOP 10` when exploring data sets to keep query times short and return only a few rows to play with.\n",
```

Comment:
Are we sure that listing TOP or maxrec does actually shorten the query time in the absence of an ra/dec coordinate constraint on the search? This hasn't been my experience when searching by various criteria without ra/dec constraints, but maybe my searches were still not restrictive enough despite listing a small number.

Reply:
Ok I just did a little more testing and it still seemed the TAP query was prohibitive even when specifying 10 without the ra/dec. Suggest updating text on this line to say something like "adding maxrec or select top 10 to further shorten query times".

Reply:
That makes a lot of sense. I did some more testing too. It only quickens the query when it has no WHERE statement to deal with. So I changed the wording to be: 
_"As demonstrated in Section 2.3.2, use `maxrec=10` or `SELECT TOP 10` when exploring data sets in order to return only a few rows to play with (this can also shorten query times for exploratory queries without WHERE statements)."_